torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1682,6 +1682,8 @@ class InsertQuantDeQuantHelper {
     is_dynamic_ = is_dynamic;
   }
 
+  void quantizeGeneralOps(Module& module);
```

Comment:
Can we have a more descriptive name for this function:  Can we call this propagateQuantDeQuant?

Reply:
Sure

Reply:
you can find the comment in quantization.h

Reply:
renaming is done in next PR